GitHub Action: "Update Editor's Copy" is failing.

It seems "Update Editor's Copy" has been failing.

https://github.com/WICG/webpackage/actions/workflows/ghpages.yml

[Log](https://github.com/WICG/webpackage/runs/4307388595?check_suite_focus=true):

> bikeshed spec loading.bs
> bash: line 1: bikeshed: command not found
> make: *** [Makefile:16: loading.html] Error 127

It seems `bikeshed` is not found.

Is there someone who can fix this?

Is this an issue of the upstream GitHub action, [i-d-template](https://github.com/martinthomson/i-d-template), or our repository setup issue?

--- a/.github/workflows/ghpages.yml
+++ b/.github/workflows/ghpages.yml
@@ -39,6 +39,13 @@ jobs:
     - name: "Build Drafts"
       uses: martinthomson/i-d-template@v1
 
+    - name: "Build Extra Pages"
+      run: |
+        pip3 install bikeshed
+        bikeshed update
+        bikeshed spec loading.bs
+        bikeshed spec subresource-loading.bs
+
     - name: "Update GitHub Pages"
       uses: martinthomson/i-d-template@v1
       if: ${{ github.event_name == 'push' }}

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-GHPAGES_EXTRA := loading.html
+GHPAGES_EXTRA := loading.html subresource-loading.html
 
 LIBDIR := lib
 include $(LIBDIR)/main.mk
@@ -11,6 +11,3 @@ else
 	git clone -q --depth 10 $(CLONE_ARGS) \
 	    -b main https://github.com/martinthomson/i-d-template $(LIBDIR)
 endif
-
-%.html: %.bs
-	bikeshed spec $<

